libraries/abstractions/transport/utest/CMakeLists.txt

```diff
@@ -1,101 +1,89 @@
-    project ("transportsecure sockets unit test")
-    cmake_minimum_required (VERSION 3.13)
+project ("transportsecure sockets unit test")
+cmake_minimum_required (VERSION 3.13)
 
 # ====================  Define your project name (edit) ========================
-    set(project_name "transport_secure_sockets")
+set(project_name "transport_secure_sockets")
 
 # =====================  Create your mock here  (edit)  ========================
 
 # list the files to mock here
-    list(APPEND mock_list
-                "${kernel_dir}/include/task.h"
-                "${kernel_dir}/include/portable.h"
-                "${common_dir}/include/iot_logging_task.h"
-                "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
-            )
+list(APPEND mock_list
+            "${abstraction_dir}/secure_sockets/include/iot_secure_sockets.h"
+        )
 # list the directories your mocks need
-    list(APPEND mock_include_list
-                "${CMAKE_CURRENT_LIST_DIR}/include"
-            )
+list(APPEND mock_include_list
+            "${CMAKE_CURRENT_LIST_DIR}/include"
+        )
 #list the definitions of your mocks to control what to be included
-    list(APPEND mock_define_list
-                LWIP_COMPAT_SOCKETS=1
-                portHAS_STACK_OVERFLOW_CHECKING=1
-                portUSING_MPU_WRAPPERS=1
-                MPU_WRAPPERS_INCLUDED_FROM_API_FILE
-           )
+list(APPEND mock_define_list
+            LWIP_COMPAT_SOCKETS=1
+            portHAS_STACK_OVERFLOW_CHECKING=1
+            portUSING_MPU_WRAPPERS=1
+            MPU_WRAPPERS_INCLUDED_FROM_API_FILE
+       )
 
 # ================= Create the library under test here (edit) ==================
 
-# The single source of truth for the transport_interface.h is in the coreMQTT repository.
-# In the FreeRTOS Device Software download page in AWS IoT Core, it is possible
-# to choose coreHTTP instead of coreMQTT. In this case, it is necessary to check
-# for the existence of the file in the downloaded software package.
-if( EXISTS ${AFR_MODULES_DIR}/coreMQTT/source )
-    set(transport_interface_dir "${AFR_MODULES_DIR}/coreMQTT/source/portable")
-elseif( EXISTS ${AFR_MODULES_DIR}/coreHTTP/source )
-    set(transport_interface_dir "${AFR_MODULES_DIR}/coreHTTP/source/portable")
-else()
-    message( FATAL_ERROR "No transport_interface.h exists for this included interface.")
-endif()
+# Include filepaths for source and include.
+include( ${AFR_ROOT_DIR}/libraries/abstractions/transport/transport_interface.cmake )
```

Comment:
(Nit)
```suggestion
include( ${AFR_MODULES_ABSTRACTIONS_DIR}/transport/transport_interface.cmake )
```

Reply:
Same for other occurrences of `${AFR_ROOT_DIR}/libraries/abstractions`

Reply:
fixed in 73655a42aa28dcc77c032db2e3ff129a2939d64e